docs/docs/manual/starting.md

```diff
@@ -80,24 +76,22 @@ This can be useful if you want to pre-select a specific number of rows from your
 
 This can also be useful if you would like to paste in a list of URLs, which you can use later to fetch the data online and build columns with. 
 
-
 #### Database (SQL)
 
 You may want to pull the latest dataset directly from an online database. This could include an online catalogue, a Wordpress or similar content management system, or a [digital repository or collection management system](https://bits.ashleyblewer.com/blog/2017/08/09/collection-management-system-collection/).
 
 OpenRefine can connect to PostgreSQL, MySQL, MariaDB, and SQLite database systems. It will automatically populate the “Port” field based on which of these you choose, but you can manually edit this if needed.
 
-To import data directly from a database online, you will need to do two things:
-
+If you have a downloaded database (`.db`) file, you can supply the path to the file on your computer directly in the “Database” field at the bottom of the form. You can leave the rest of the fields blank.
```

Comment:
It should be clear that this is specific to SQLite: connecting to a SQLite database can only be done by supplying the path to such a file, and SQLite is the only database type where you provide the path to a file.

Reply:
This is marked as resolved but I'm not clear it has been resolved? It feels like we should be specific that it is a SQLite database file that is being referenced here - I don't think just saying `a downloaded database ('.db') file` is specific enough.

Generally I think resolutions need to be clear:
* Change made that directly addresses the comment - I think the change speaks for itself and is fine
* Decision to not make any changes, or change in a different way to what was suggested  - I think worth noting this is the action happening if it isn't going to be obvious to the reviewer
* No decision right now, and will come back for later consideration - again worth noting this action and I'd encourage an issue as a way of tracking although it may not always be necessary